fluent-plugin-papertrail.gemspec

```diff
@@ -4,7 +4,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name          = "fluent-plugin-papertrail"
-  spec.version       = "0.1.3"
+  spec.version       = "0.1.4"
```

Comment:
Why are you incrementing PATCH here instead of MINOR? Per the [Semver guidelines](https://semver.org), you should be incrementing MINOR when you're adding backwards-compatible functionality and PATCH when you're fixing bugs. I'd make this `0.2.3`